test/test_respondent_controller.py

```diff
@@ -101,6 +101,26 @@ def test_get_respondent_by_email_returns_correct_representation(self):
         self.assertEqual(response['sampleUnitType'], self.mock_respondent['sampleUnitType'])
         self.assertEqual(response['telephone'], self.mock_respondent['telephone'])
 
+    def test_update_respondent_details_success(self):
+        self.populate_with_respondent(respondent=self.mock_respondent_with_id)
+        respondent_id = self.mock_respondent_with_id['id']
+        payload = {
+            "firstName": "John",
+            "lastName": "Snow",
+            "telephone": "07837230942"
+            }
+        self.change_respondent_details(respondent_id, payload, 200)
```

Comment:
Can we do the assertion here?

Something like 

```
resp = self.change_respondent_details(respondent_id, payload)
assert resp.status_code == 200
```

Reply:
@benjefferies I agree we should be asserting but the status check is already done in the helper function. We should probably be asserting the contents returned against expected instead.

```
data =  self.change_respondent_details(respondent_id, payload, 200)
self.assertEqual(data, 'Successfully updated respondent details')
```